### DISCUSSION TITLE
Magnetic susceptibility

### DISCUSSION THREAD

**Discussion by AprilSweettooth** [Part-II-Computational-Physics/.github#58] [Q&A] [answered]
Do we consider $\beta$ or $\beta J$ in this question? It seems the formula derived is in the format of $\beta$, but we are asked to calculate a wide range of $\beta J$ without a known J.

**Comment by DLHvdH** [comment, marked as answer]
That's a good question! It seems you spotted a mistake of mine. Do not worry about the prefactor. The goal is to investigate the variance of the magnetisation. As you might have noticed, the magnetisation itself also doesn't have a prefactor beta, even though you would expect one. The magnetisation has been normalised to give 0 for no magnetisation and -1 or 1 for full magnetisation.
